common/src/main/java/io/netty/util/internal/logging/Slf4JLoggerFactory.java

```diff
@@ -30,6 +30,8 @@
     @SuppressWarnings("deprecation")
     public static final InternalLoggerFactory INSTANCE = new Slf4JLoggerFactory();
 
+    public static final InternalLoggerFactory INSTANCE_WITH_NOP_CHECK = new Slf4JLoggerFactory(true);
```

Comment:
Make the instance package private?

Reply:
have changed the instance to package private.